src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Altinn.Platform.Storage.Interface.csproj

```diff
@@ -12,7 +12,7 @@
   <PropertyGroup>
     <!-- NuGet package properties -->
     <PackageId>Altinn.Platform.Storage.Interface</PackageId>
-    <PackageVersion>2.5.10</PackageVersion>
+    <PackageVersion>2.5.11</PackageVersion>
```

Comment:
should probably have bumped the minor version as the type for the list content of presentation fields was changed. Just a heads up for the next time a change like this is made.